yarn-project/end-to-end/src/e2e_account_contract.test.ts

```diff
@@ -23,7 +23,7 @@ import { PublicKey, TxStatus } from '@aztec/types';
 import { privateKey2 } from './fixtures.js';
 import { setup } from './utils.js';
 
-describe('e2e_account_contract', () => {
+describe.skip('e2e_account_contract', () => {
```

Comment:
Looks fine, just wondered if this is intentional?

Reply:
It is, I'm reenabling it in #1054